src/frontend/src/components/SearchAndDisplayResults.tsx

```diff
@@ -33,52 +41,25 @@ export default function Searching() {
       [loading, hasMore]
    );
 
-   /*const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-      setQuery(e.target.value);
-      setPageNum(1);
-      setPress(false);
-   };
-
-   const handlePress = (e: KeyboardEvent<HTMLInputElement>) => {
-      // POST request when enter is pressed
-      if (e.key === 'Enter') {
-         e.preventDefault();
-         setPress(true);
+   const handleSubmit = (
```

Comment:
Search is not the same thing as getting apartments to display.

Reply:
explain